checks.d/mongo.py

```diff
@@ -829,7 +829,7 @@ def total_seconds(td):
             pass
 
         dbnames = cli.database_names()
-        GAUGE(self, 'mongodb.dbs', len(dbnames), tags=tags)
+        self.gauge('mongodb.dbs', len(dbnames), tags=tags)
```

Comment:
I think you might be right that line might be problematic. However, it affects a lot of the check, so it should be separate from this change.